### PR TITLE
Added support for Hapi.JS v17

### PR DESCRIPTION
**Pull request by tejzpr**
Added support for Hapi.JS v17 and bumped package major version.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,3452 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (nelsonic) [inline yarn.lock:1]: we _probably_ don't need this file ...

**Review by nelsonic — APPROVED** [review]
@tejzpr thank you _very_ much for taking the time to create this Pull Request updating the code/examples/tests to Hapi v.17 🎉

File: yarn.lock (hunk 1 above)
Comment (tejzpr) [reply yarn.lock:1]: https://github.com/yarnpkg/yarn/issues/1583 The yarn.lock file was supposed to be committed along with the package.json file.

**Comment by tejzpr** [comment]
Yay!  🎉

**Pull request merged**